firebase_admin/project_management.py

```diff
@@ -678,28 +646,20 @@ def add_sha_certificate(self, app_id, certificate_to_add):
         sha_hash = _check_not_none(certificate_to_add, 'certificate_to_add').sha_hash
         cert_type = certificate_to_add.cert_type
         request_body = {'shaHash': sha_hash, 'certType': cert_type}
-        self._make_request('post', path, app_id, 'App ID', json=request_body)
+        self._make_request('post', path, json=request_body)
 
     def delete_sha_certificate(self, certificate_to_delete):
         name = _check_not_none(certificate_to_delete, 'certificate_to_delete').name
         path = '/v1beta1/{0}'.format(name)
-        self._make_request('delete', path, name, 'SHA ID')
+        self._make_request('delete', path)
+
+    def _make_request(self, method, url, json=None):
+        body, _ = self._body_and_response(method, url, json)
+        return body
 
-    def _make_request(self, method, url, resource_identifier, resource_identifier_label, json=None):
```

Comment:
This is a somewhat minor point: Does `_utils.handle_platform_error_from_requests` raise an error that includes the request? If not, then we might be losing some helpful information in the error.

Reply:
One can get to the outgoing `requests.Request` object as follows:

```
try:
  method_call()
except exceptions.FirebaseError as error:
  req = error.http_response.request
  print(req.method, req.url, req.body)
```

The raised error also includes the original `requests.error.RequestError`. Among all this, and the exception stacktrace, developers should have enough details to implement fine-grained error handling.

Reply:
Ah, sounds good.